(153) Start page content

leeky opened the pull request:
<img width="846" alt="start-page-new" src="https://user-images.githubusercontent.com/3166/31711326-a67b6b5e-b3ef-11e7-97f6-6493551b2734.png">

leeky committed:
(chore) Update start page copy

dgmstuart reviewed (changes requested):
I hate to be that guy, but does this maybe "need more whitespace"? On the example on gov.uk the page is broken up into 3 distinct sections:
https://www.gov.uk/service-manual/design/start-pages

At the moment the start page feels like all one block, which might feel intimidating.

Perhaps some more space between title, "what this is" and "before you start" would help with that?

Also in the "before you start" section the spacing between the title and the two paras feels odd to me. In the Gov.uk example these are evenly spaced.

leeky committed:
(chore) Fix header alignment

 - Fix alignment of Hackney logo in header
 - Increase width of green line in border, to balance with heading text
   font weight
 - Make header take us less room on mobile

leeky committed:
(chore) Typography fixes

 - Fix use of Droid Sans in the font stack. It was being loaded, but
   overriden by the (unavailable) GOVUK font 'GDS Transport'
 - Increase font size of headings
 - Increase line height of paragraphs, to increase readability

leeky committed:
(chore) Add "style guide" page with example styling

This is mainly here to preview the various styles used across the site
in one place for quick reference.

Also, included the missing `panels` component.

leeky committed:
(chore) Increase whitespace throughout

Instead of using an arbitary amount of spacing, use the `$gutter` and
`$gutter-half` variables defined in the GOVUK frontend toolkit.

leeky committed:
(fix) Buttons are always Hackney green

Previously, buttons with *visited* links would be GOVUK green instead.

leeky commented:
I've given the styling a good kick - increasing whitespace, using the correct font etc, and added some commits to this branch:

![report-a-repair](https://user-images.githubusercontent.com/3166/31766299-14fd51a8-b4bf-11e7-90e9-beedbf14f335.png)

I've tested in with an older iPhone with BrowserStack, and it seems happier on there too.

![report-a-repair-mobile](https://user-images.githubusercontent.com/3166/31766934-2aebd06e-b4c1-11e7-819f-81750651b481.png)

leeky commented:
@dgmstuart Should be ready for another look.
